db/migrate/20190726175042_add_case_insensitive_index_to_tags.rb

```diff
@@ -15,7 +15,13 @@ def up
       Tag.where(id: redundant_tag_ids).in_batches.delete_all
     end
 
-    safety_assured { execute 'CREATE UNIQUE INDEX CONCURRENTLY index_tags_on_name_lower ON tags (lower(name))' }
+    begin
+      safety_assured { execute 'CREATE UNIQUE INDEX CONCURRENTLY index_tags_on_name_lower ON tags (lower(name))' }
+    rescue ActiveRecord::StatementInvalid
+      remove_index :tags, name: 'index_tags_on_name_lower'
+      raise
```

Comment:
Why does this raise a StatementInvalid error but others are raising CorruptionError?

Reply:
It's not pointing at the documentation because it's a much older migration which is not handled by the maintenance script. It's very unlikely people run into issues running this migration, but I wanted to add the `remove_index` to at least ensure the effects of the migration would be reverted if it ran into an issue.